tests/unit_tests/test_unit.py

```
@@ -2522,3 +2522,124 @@ def dummy(*args, **kwargs):
         assert res.exit_code == 0
         assert "Error" not in res.stderr
         assert "ok" in res.stdout
+# issue 142- mgf scan number tracking
+
+def test_mgf_scan_index_built_correctly(mgf_small_with_scans, tmp_path):
```

Ruff (0.15.6)

[warning] Unused function argument: `tmp_path`

(ARG001)